Sources/SwiftyUserDefaults.swift

```diff
@@ -133,14 +133,22 @@ public extension UserDefaults {
             return proxy
         }
         set {
+            
+            guard let newValue = newValue else {
+                removeObject(forKey: key)
+                return
+            }
+            
             switch newValue {
-            case let v as Int: self.set(v, forKey: key)
+                
+            // @warning This should always be on top of Int because a cast
+            // from Double to Int will always succeed.
```

Comment:
That's interesting… Has that changed in Xcode 3? Because I'm fairly confident the way it was before worked fine with Doubles...